### DISCUSSION TITLE
How to add images and tables to blog?

### DISCUSSION THREAD

**Discussion by 0ldcamel** [skills/.github#673] [GitHub Pages]
I’m working on my blog, following the instructions. So far my blog works fine but I am running into some issues with adding images and tables. I’ve tried searching for answers, but it hasn’t resolved the issue.

If anyone has experience with this or could point me in the right direction, I’d really appreciate your help!

**Comment by Roject-CN** [comment]
Bro, can you describe some details about your issue? 
I encountered the same situation as similar as yours.
It's already 2025. Have you solved your problems?

**Reply by 0ldcamel** [reply]
I can add images now but not tables yet.
